decidim-pages/config/locales/en.yml

```diff
@@ -1,9 +1,9 @@
 ---
 en:
-  components:
-    page:
-      name: Page
   decidim:
+    features:
+      pages:
+        name: Pages
```

Comment:
We might need a way to handle singularized names for components. The page for the meetings list has a singular name:

http://decidim-design.marsbased.com/phase1_meetings/ ("Trobada")